include/coap3/coap_internal.h

```diff
@@ -35,6 +35,14 @@
 # include <assert.h>
 #endif
 
+#ifdef HAVE_INTTYPES_H
+#include <inttypes.h>
+#else /* ! HAVE_INTTYPES_H */
+#ifndef PRIx32
+#define PRIx32 "x"
```

Comment:
There was `PRIu32` (apart from `PRIx32`) defined in source files as well. Did you omit it on purpose or just missed it?

Reply:
Oops - missed it.  Will fix it - thanks.